src/EnergyPlus/AirflowNetwork/src/Solver.cpp

```diff
@@ -9488,7 +9444,7 @@ namespace AirflowNetwork {
 
         for (std::size_t i = 0; i < linkReport.size(); ++i) {
             auto &r(linkReport[i]);
-            auto &s(AirflowNetworkLinkSimu[i]);
+            auto const &s(AirflowNetworkLinkSimu[i]);
```

Comment:
This is neither here nor there (and I know you didn't do this @rraustad) but I have to say that I hate constructor notation for non-object variables. What is wrong with `=`? Are you really calling a constructor on a reference here? You are just assigning a pointer, not creating a copy of the object. Are you just trying to show people that you've heard of constructor notation? In defense of this idiom, at least it's not array/list constructor notation, i.e., `{}`. 🤮